### PR TITLE
feat: create chiselled Nginx ROCK

### PR DESCRIPTION
**Pull request by cjdcordeiro**
- [x] I have signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] I am following the [CONTRIBUTING](https://github.com/canonical/oci-factory/blob/main/CONTRIBUTING.md) guidelines

*Ping the @canonical/rocks team.*

---

## Description
Add an internal chiselled Nginx ROCK based on the existing ubuntu/nginx. Unlike the existing one, this is a chiselled ROCK and not a Docker image.

It will be kept internal (GHCR only) for now, since we can't yet build the ROCKs for multiple architectures.

### CODE REVIEW

File: oci/nginx/image.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 1
+
+upload:  
+  - source: https://git.launchpad.net/~ubuntu-docker-images/ubuntu-docker-images/+git/nginx
```
Comment (SamirPS) [inline oci/nginx/image.yaml:4]: can't we use more ssh url?

Comment (cjdcordeiro) [reply oci/nginx/image.yaml:4]: more? like more than 1? Why?

Comment (SamirPS) [reply oci/nginx/image.yaml:4]: oh no like instead of https use git+ssh

Comment (cjdcordeiro) [reply oci/nginx/image.yaml:4]: but then you'd need to add credentials to the OCI factory

Comment (SamirPS) [reply oci/nginx/image.yaml:4]: yeah true

**Review by SamirPS — APPROVED** [review]

**Pull request merged**